mne_bids/commands/mne_bids_raw_to_bids.py

```diff
@@ -68,15 +68,15 @@ def run():
         parser.error('Arguments missing. You need to specify at least the'
                      'following: --subject_id, --task, --raw, --bids_root.')
 
-    bids_basename = BIDSPath(
+    bids_path = BIDSPath(
         subject=opt.subject_id, session=opt.session_id, run=opt.run,
-        acquisition=opt.acq, task=opt.task)
+        acquisition=opt.acq, task=opt.task, root=opt.bids_root)
     raw = _read_raw(opt.raw_fname, hpi=opt.hpi, electrode=opt.electrode,
                     hsp=opt.hsp, config=opt.config,
                     allow_maxshield=opt.allow_maxshield)
     if opt.line_freq is not None:
         raw.info['line_freq'] = opt.line_freq
-    write_raw_bids(raw, bids_basename, opt.bids_root, event_id=opt.event_id,
+    write_raw_bids(raw, bids_path, event_id=opt.event_id,
```

Comment:
should we do a deprecation cycle here? I guess other people might be using these functions? Or do we completely break backwards compatibility this release?

Reply:
I'm okay breaking it :p, but up to you all.

Reply:
> should we do a deprecation cycle here? I guess other people might be using these functions? Or do we completely break backwards compatibility this release?

I guess we're breaking pretty much everything already… I mean, we even removed `make_bids_basename`… NO user code will work anymore ;)